package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "io.appium.settings",
-  "version": "2.13.0",
+  "version": "2.13.1",
```

Comment:
should the version be bumped like this?

Reply:
Nope. That was me trying out the version script.